Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -634,7 +634,8 @@ u32 GetTicksPerField()
 
 static void BeginField(FieldType field)
 {
-	bool interlaced_xfb = ((m_PictureConfiguration.STD / m_PictureConfiguration.WPL)==2);
+	bool potentially_interlaced_xfb = ((m_PictureConfiguration.STD / m_PictureConfiguration.WPL)==2); // Could we fit a second line of data in the stride?
```

Comment:
You could add some whitespace there around the `==` while you're at it.
And personally, I'd put the comment in the line before; not at the end.